cmd/mount/mount.go

```diff
@@ -73,6 +74,29 @@ func mountOptions(device string) (options []fuse.MountOption) {
 // report an error when fusermount is called.
 func mount(f fs.Fs, mountpoint string) (*vfs.VFS, <-chan error, func() error, error) {
 	fs.Debugf(f, "Mounting on %q", mountpoint)
+
+	// Check is folder is empty
+	if !mountlib.AllowNonEmpty {
+		fp, fpErr := os.Open(mountpoint)
+		if fpErr != nil {
+			fp.Close()
```

Comment:
The build is complaining about you not checking the error from close.

If you factor this into a function then you can write

    defer fs.CheckClose(fp, &err)

after the open (you'll have to name the error return parameter).